### ISSUE TITLE
Add prompt for CPU governor selection

### ISSUE THREAD

**Issue by damian101** [Frogging-Family/linux-tkg#711]
The current linux-tkg default "ondemand" causes huge performance degredation on some CPUs (my AMD EPYC for example) compared to mainline default "schedutil". Which is why I think this potentially very important setting should be more exposed, by having a prompt for it by default.

**Comment by mabod** [comment]
This is interesting. Can you share some more info about the performance degradation? Any article or benchmark you can share?

**Comment by damian101** [comment]
> This is interesting. Can you share some more info about the performance degradation? Any article or benchmark you can share?

I have to take this back. I observed this half a year ago, so I sadly don't remember the exact details from back then, only that ondemand performed much worse than schedutil, which performed almost as good as performance, in some benchmark, but now I did some benchmarks again and found no difference for Geekbench 6 (to my surprise), but a huge performance benefit in Geekbench 5 (which I used back then) for schedutil over ondemand. This is definitely connected to my choice of CPU (dual-socket first gen Epyc, maybe it's just the many cores?). I then performed additional benchmarks (with phoronix-test-suite) and found no signficant differences for most workloads, like compiling and 3D rendering, but some very significant differences for stuff like video encoding, where I got almost twice the performance with ondemand over schedutil in one workload (x265 1080p). 
So, everything's probably just fine the way it is (adding the prompt might still be a good idea, not sure). Geekbench 5 is just a bad benchmark. Closing this for now.
